tests/System/ImportTest.php

```diff
@@ -103,7 +103,7 @@ public function getApiTestsToRun()
         $apiToTest = [];
         $apiNotToTest = [];
 
-        $config = require_once PIWIK_INCLUDE_PATH . '/plugins/GoogleAnalyticsImporter/config/config.php';
+        $config = require PIWIK_INCLUDE_PATH . '/plugins/GoogleAnalyticsImporter/config/config.php';
```

Comment:
What's the reason for this change?

Reply:
so if you try to run the test locally this breaks, reason being config already loaded and for next time require_once returns true not the content and it breaks the test, to resolve it changed to `require`